README.md

```diff
@@ -183,4 +188,67 @@ Developer Student Club Universidade Eduardo Mondlane é uma comunidade ou clube
 4. [Instagram](https://www.instagram.com/dscuem/)
 </details>
 
+
+---
+### Maputo Frontenders
+
+Somos uma comunidade de desenvolvedores com foco em partilhar conhecimento sobre as tecnologias e boas praticas de desenvolvimento frontend.
+
+<details>
+  <summary>Ver detalhes</summary>
+  Via de Comunicação
+
+1. [ Maputo Frontenders Community ](https://bit.ly/MptFrontenders)
+2. [Whatsapp](https://bit.ly/MFrontendersWhatsApp)
+3. [Twitter](https://bit.ly/MFrontendersTwitter)
+4. [Instagram](https://bit.ly/InstaMptFrontenders)
+5. [LinkedIn](https://bit.ly/MFrontendersLinkedIn)
+6. [YouTube](https://bit.ly/YTmptFrontenders)
```

Comment:
I think the order shouldn't really matter?

```suggestion
- [ Maputo Frontenders Community ](https://bit.ly/MptFrontenders)
- [Whatsapp](https://bit.ly/MFrontendersWhatsApp)
- [Twitter](https://bit.ly/MFrontendersTwitter)
- [Instagram](https://bit.ly/InstaMptFrontenders)
- [LinkedIn](https://bit.ly/MFrontendersLinkedIn)
- [YouTube](https://bit.ly/YTmptFrontenders)
```

Reply:
Doesn´t matter